components/Accordion.js

```diff
@@ -0,0 +1,195 @@
+    },
+    {
+      id: 4,
+      isOpen: false,
+      title: "What is a good place to find out more about dog ownership?",
+      content: `Here you get a good overview of some basic things to keep in mind (in German and English):`,
+      links: [
+        {
+          id: 1,
+          url: "https://www.tierschutzbund.de/tiere-themen/haustiere/hunde",
+          name: "tierschutzbund.de",
+        },
+        {
+          id: 2,
+          url: "https://www.four-paws.org/campaigns-topics/animals/dog",
+          name: "four-paws.org",
+        },
+      ],
+    },
+
+    {
+      id: 5,
+      isOpen: false,
+      title: "Should I immediately go get my matching dog now?",
+      content: `No! Dog ownership comes with a lot of responsibility. A dog can live up to 15 years and you should think well about whether you can provide the care the dog needs for that long, financially, emotionally and otherwise.`,
+      links: [],
+    },
+    {
+      id: 6,
+      isOpen: false,
+      title: "How can I know more about a dog breed?",
+      content: `You can view all dog breeds on the "All dogs page" that can be found in the hamburger navigation, and there you can search for a breed. `,
+      links: [
+        {
+          id: 1,
+          url: "/dogs",
+          name: "All dogs page",
+        },
+      ],
+    },
+    {
+      id: 7,
+      isOpen: false,
+      title: "Can I decide on a dog with the app alone?",
+      content: `No. While we carefully put together our algorithm, the seven questions asked are not enough to find out if it's the right dog for you. There are many factor to consider such as who you live with, how much space you have and how much training you can provide the dog in order to fully understand if a dog could live with you happily.`,
+      links: [],
+    },
+  ]);
+
+  // Logic
+  function handleToggle(id, e) {
+    e.preventDefault();
+    setOpenIndex(id);
+    setAccordionData((accordionItems) => {
+      const info = accordionItems.find((info) => info.id === id);
+
+      if (info) {
+        return accordionItems.map((info) => {
+          if (info.id === id) return { ...info, isOpen: !info.isOpen };
+          else return { ...info, isOpen: false };
+        });
+      }
+
+      return [...accordionItems, { id, isOpen: false }];
+    });
+  }
+  return accordionData.map(({ id, title, content, links, isOpen }) => (
+    <StyledDetails key={id} open={isOpen}>
+      <StyledSummary onClick={(e) => handleToggle(id, e)}>
+        {title}
+
+        {id === openIndex && (
+          <StyledIcon
+            src="/chevron-up.svg"
+            alt="Chevron up arrow"
+            width={29}
+            height={29}
+            blurDataURL="data:..."
+          />
+        )}
+        {id != openIndex && (
+          <StyledIcon
+            src="/chevron-down.svg"
+            alt="Chevron down arrow"
+            width={29}
+            height={29}
+            blurDataURL="data:..."
+          />
+        )}
```

Comment:
by changing your code to this, you could save some lines:

```suggestion
        <StyledIcon
          src={id === openIndex ? "/chevron-up.svg" : "/chevron-down.svg"}
          alt={id === openIndex ? "Chevron up arrow" : "Chevron down arrow"}
          width={29}
          height={29}
          blurDataURL="data:..."
        />
```